From 43542cc07ee110abe25a1e176238fd6d921d593b Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Mon, 9 Aug 2021 23:07:12 +0200
Subject: Missing include

(for std::unique_ptr, with recent libstdc++ 12 trunk)

Change-Id: I61b7823dd740ea7cdfe0d7403a50ac73b24d1c4e
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/120229
Tested-by: Jenkins
Reviewed-by: Stephan Bergmann <sbergman@redhat.com>
---
 shell/source/backends/kf5be/kf5backend.cxx | 2 ++
 1 file changed, 2 insertions(+)

(limited to 'shell/source/backends/kf5be/kf5backend.cxx')

diff --git a/shell/source/backends/kf5be/kf5backend.cxx b/shell/source/backends/kf5be/kf5backend.cxx
index 47c829dc5f9b..e448c124d88a 100644
--- a/shell/source/backends/kf5be/kf5backend.cxx
+++ b/shell/source/backends/kf5be/kf5backend.cxx
@@ -19,6 +19,8 @@
 
 #include <sal/config.h>
 
+#include <memory>
+
 #include <QtWidgets/QApplication>
 
 #include <boost/noncopyable.hpp>
-- 
cgit v1.2.1

